CHANGELOG.md

```diff
@@ -18,6 +18,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [Dimitris Koutsogiorgas](https://github.com/dnkoutso)
   [#9423](https://github.com/CocoaPods/CocoaPods/issues/9423)
 
+* Option to lint a specified set of test_specs  
```

Comment:
should probably be an enhancement? If it is should it be against master instead?

Reply:
It's arguable.  The case for it being a bugfix is that `pod lib lint` is unusable for testing some podspecs is some configurations without it.

Our use case is wanting to run unit tests for external PRs, but not integration tests that require secrets.

I'll go with whatever you think is best.

Reply:
Rebased and updated CHANGELOG